source/oleacc.py

```diff
@@ -313,15 +313,15 @@ def AccessibleObjectFromEvent_safe(hwnd, objectID, childID, timeout=2):
 
 def WindowFromAccessibleObject(pacc):
 	"""
-	Retreaves the handle of the window this IAccessible object belongs to.
+	Retrieves the handle of the window this IAccessible object belongs to.
 	@param pacc: the IAccessible object who's window you want to fetch.
 	@type pacc: POINTER(IAccessible)
 	@return: the window handle.
 	@rtype: int
 	"""
-	hwnd = c_int()  # noqa: F405
+	hwnd = HWND()  # noqa: F405
```

Comment:
please fix the F405 error for this line, and update the docstring/type hints for this function

Reply:
I'm afraid I won't have time to do this. Note that this is a fixup for #18834.

Reply:
all you need to do is:
- add `from ctypes import HWND` to the top of file
- remove `noqa: F405` from this line
- move the type information from the docstring into type hints

Reply:
How to typehint the IAccessible pointer?